CONTRIBUTING.md

```diff
@@ -4,22 +4,74 @@ First off, thanks for taking the time to contribute!
 
 The following is a set of guidelines and helpful information for contributing to the 3D Artist's Handbook, which is hosted on [GitHub](https://github.com/sidney-eliot/3d-artists-handbook). These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
 
-#### Table Of Contents
-
-ToDo
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [What Should I Know Before I Get Started?](#what-should-i-know-before-i-get-started)
+   - [Prerequisites](#prerequisites)
+   - [Choosing What to Work On](#choosing-what-to-work-on)
+- [How Can I Contribute?](#how-can-i-contribute)
+   - [Test Locally Before You Push](#test-locally-before-you-push)
+- [Styleguides](#styleguides)
+   - [Handbook Pages Styleguide](#handbook-pages-styleguide)
+      - [General](#general)
+      - [Specific](#specific)
+- [Additional Note](#additional-notes)
 
 ## Code of Conduct
+
 This project and everyone participating in it is governed by the [3D Artist's Handbook Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
 
-## What should I know before I get started?
+## What Should I Know Before I Get Started?
 
-ToDo
+### Prerequisites
+
+1. You will need to have [Node.js](https://nodejs.org/en/download/package-manager), and thus `npm` available on your
+   terminal of choice.
+
+   > [!NOTE]
+   > Installing the latest version should work without much problem. However, just in case as a reminder, as of 
+   > 2024-12-10, Quartz 4 needs _at least_ Node v20 and up, as well as `npm` v9.3.1 and up to work.
+   >
+   > By the time you read this, the information contained here may be outdated, for up to date information on Quartz's
+   > dependencies, please consult Quartz's own documentation, which should be found [here](https://quartz.jzhao.xyz/).
+
+2. Then, clone this repository in a directory of your choosing.
+
+3. Change directories (`cd`) to be inside the repository you just cloned.
+
+4. Inside the repository, execute the following commands:
+
+   ```shell
+   npm i
+   npx quartz build --serve
+   ```
+
+5. Now, open a new browser window with the address: [https://localhost:8080](https://localhost:8080)  
+   Note: If the above commands worked properly, you should see a prompt with this address as a result.
+
+### Choosing What to Work On
+
+Before writing anything, please try to familiarize yourself with the [styleguides](#styleguides) below.
```

Comment:
That's a good point, I wasn't entirely sure where to include this. 🤔
I saw it as something a contributor should consult right before or while writing.

It may indeed be wiser to place it in the "Prerequisites" section. Do you agree?